### PR TITLE
Blivet API demo creating encrypted LVM on two drives.

### PR DESCRIPTION
**Pull request by dwlehman**
This demo was written against blivet-2.1.8. Some modifications may be needed to run against other versions.

### CODE REVIEW

File: demo-1-blivet [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        device = blivet_obj.devicetree.resolve_device(path)
+        if device is None:
+            sys.stderr.write("ERROR: failed to locate device '%s'\n" % path)
+            sys.stderr.write("Here's what we did find:\n%s" % str(blivet_obj.devicetree))
+            sys.exit(1)
+
+        # wipe existing metadata from the device
+        blivet_obj.devicetree.recursive_remove(device, remove_device=False)
+        if cleanup:
+            continue
```
Comment (vpodzime) [inline demo-1-blivet]: Could you please split the function into two -- one doing the setup and another one doing the cleanup? I'm afraid people would find this combined one confusing.

File: demo-1-blivet [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                                    fmt_type="luks",
+                                    fmt_args={"passphrase": LUKS_PASSPHRASE})
+        blivet_obj.create_device(data_lv)
+
+        luks_dev = LUKSDevice("luks-%s" % data_lv.name,
+                              parents=data_lv,
+                              fmt=get_format("xfs", label=DATA_LABEL))
+        blivet_obj.create_device(luks_dev)
+
+    blivet_obj.do_it()
```
Comment (vpodzime) [inline demo-1-blivet]: I think it's worth commenting what ``do_it()`` does, not matter how much it's name speaks for itself. :)

**Review by vpodzime — APPROVED** [review]
Looks good to me otherwise.

File: demo-1-blivet (hunk 2 above)
Comment (dwlehman) [reply demo-1-blivet]: Yeah, good idea.

File: demo-1-blivet (hunk 1 above)
Comment (dwlehman) [reply demo-1-blivet]: Will do.

**Comment by vpodzime** [comment]
Looks good to me and works just fine. :)

**Pull request merged**